### PR TITLE
add alias `freeze_requirements()` for `generate_requirements()`

### LINKED ISSUE #85: freeze wrapper of gen req function to mimic PIP naming
`generate_requirements()` is essentially `pip freeze > requirements.txt`

To make this more obvious to python users coming to this package a wrapper of `generate_requirements()` should be made that includes "freeze" in the name.  We discussed offline to keep `generate_requirements()` as well since this naming might be more user-friendly to a pure R user who has never used `pip freeze`

### PR DESCRIPTION
**Pull request by AdamSpannbauer**
closes #85 (not sure where i got 81 from when naming the branch)

### CODE REVIEW

File: R/gen_requirements.R [hunk 1]
```diff
@@ -85,3 +85,18 @@ generate_requirements = function(glob_paths = "*.R",
 
   write_requirements_file(requirements_vector, output_path)
 }
+
+
+#' @rdname generate_requirements
+#' @export
+freeze_requirements = function(glob_paths = "*.R",
```
Comment (bmewing) [inline R/gen_requirements.R:92]: Just making sure, do we want to keep `*_requirements` for all the function names? vs just `freeze()`

**Review by bmewing — COMMENTED** [review]
I guess I have to type something here?

File: R/gen_requirements.R (hunk 1 above)
Comment (AdamSpannbauer) [reply R/gen_requirements.R:92]: I'm open to either or even `rip_freeze()` (only partially joking).

Comment (bmewing) [reply R/gen_requirements.R:92]: we could rename all of them to `rip_*` and explain in the package the reason

Comment (AdamSpannbauer) [reply R/gen_requirements.R:92]: requirements installs packages

**Pull request merged**